src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -65,23 +68,31 @@ public void startScan() {
         }
 
         project.getMessageBus().syncPublisher(ApplicationEvents.ON_SCAN_LOCAL_STARTED).update();
-        ExecutorService executor = Executors.newFixedThreadPool(3);
-        try {
-            ScanLogic scanLogic = createScanLogic();
-            refreshScanners(scanLogic, executor);
-            NavigationService.clearNavigationMap(project);
-            for (ScannerBase scanner : scanners.values()) {
-                try {
-                    scanner.asyncScanAndUpdateResults();
-                } catch (RuntimeException e) {
-                    logError(Logger.getInstance(), "", e, true);
+        Thread currScanThread = new Thread(() -> {
```

Comment:
Disclaimer - Assaf and I agreed to leave it Thread, but may change it to a Backgroundable task after the release